Plugins.cpp

```diff
@@ -52,8 +52,20 @@ PluginInfo::PluginInfo() {
 }
 
 Plugins::Plugins(QObject *p) : QObject(p) {
-	qsSystemPlugins = QLatin1String("plugins/");
-	qsUserPlugins = QLatin1String("./");
+	// Current directory plugins
+	qsCurrentDirectoryPlugins = QDir::currentPath();
+	// System plugins directory
+	QDir qdSystemPluginsDir_x64("C:/Program Files/Mumble/Plugins");
+	QDir qdSystemPluginsDir_x86("C:/Program Files (x86)/Mumble/Plugins");
```

Comment:
Yes, I know, I already thought about it :wink: 
> it would be nice to have a way to retrieve Mumble's installation path... Maybe we could tell Mumble installer to create a file in %AppData%\Roaming\Mumble that contains the absolute location?